index.html

```diff
@@ -214,12 +214,12 @@ <h3>Extended Privacy</h3>
                         <img src="/img/privacy_written_in_tiles.jpg" class="img-responsive" alt="privacy spelled" />
                     </div>
                     <div class="feature-description">
-                        <p><strong>We protect your privacy</strong></p>
+                        <p><strong>Friends of your privacy</strong></p>
                         <ul>
-                            <li>No proprietary Google APIs</li>
-                            <li>Only send information which are required to get video and channel details</li>
-                            <li>Decide what information we keep</li>
-                            <li>We store everything offline on your device</li>
+                            <li>No Google APIs</li>
+                            <li>Only send info required to get video and channel details</li>
+                            <li>Decide what info is kept</li>
```

Comment:
I really dislike using info here, it sounds so informal imo

Reply:
Good call